### PR TITLE
chore(starters): add gatsby-starter-medium

### PR DESCRIPTION
**Pull request by s-kris**
A gatsby starter blog as close as possible to medium.

Live Demo: https://dazzling-heyrovsky-62d4f9.netlify.com

### CODE REVIEW

**Comment by s-kris** [comment]
> Danger run resulted in 1 warning; to find out more, see the [checks page](https://github.com/gatsbyjs/gatsby/runs/140027319).
> 
> Generated by 🚫 [dangerJS](https://github.com/danger/danger-js/)

@gatsbyjs/ecosystem The warning was for a different repo than mine, please correct it. 


```
    {
      "url": "https://gatsby-starter-material-business-markdown.netlify.com/",
      "repo": "https://github.com/ANOUN/gatsby-starter-material-business-markdown/"
    }

    Errors:
        "repo" with value "https://github.com/ANOUN/gatsby-starter-material-business-markdown/" fails to match the required pattern: /^https?://github.com/[^/]+/[^/]+$/
        "description" is required
        "tags" is required
        "features" is required
```

File: docs/starters.yml [hunk 1]
```diff
@@ -2820,3 +2820,13 @@
     - Styling with SCSS
     - Offline support
     - Web App Manifest
+- url: https://dazzling-heyrovsky-62d4f9.netlify.com/
+  repo: https://github.com/s-kris/gatsby-starter-medium
+  description: A GatsbyJS starter blog as close as possible to medium.
+  tags:
+    - Markdown
+    - Styling:CSS-in-JS
+    - Medium
```
Comment (amandeepmittal) [inline docs/starters.yml]: Medium is not an available tag, please remove it.

Comment (s-kris) [reply docs/starters.yml]: Removed, thanks!

**Comment by s-kris** [comment]
> Danger error was fixed, sorry for that.
> 
> Please address @amandeepmittal comment and we can merge this ;)

done, thanks! :)

**Pull request merged**